eventio/event_io_file.py

```diff
@@ -0,0 +1,65 @@
+import struct
+from .object_header import make_ObjectHeader
+from io import BytesIO
+
+class EventIOObject:
```

Comment:
I see ... makes sense, since I didn't explain anything anywhere.

I wrote an explanation, but it got too long for a comment here.
So I put it further down.